avengers/services/__init__.py

```diff
@@ -0,0 +1,23 @@
+from sendgrid import Mail, SendGridAPIClient
+from python_http_client.exceptions import BadRequestsError, UnauthorizedError
+
+from avengers.config import settings
+from avengers.presentation.exceptions import FailedToSendEmail
+
+
+def send_mail(address: str, title: str, content: str):
```

Comment:
code in init is not recommended. please move this another place named like utils.py